src/main/java/com/algorand/algosdk/v2/client/model/ErrorResponse.java

```diff
@@ -12,7 +11,7 @@
 public class ErrorResponse extends PathResponse {
 
     @JsonProperty("data")
-    public HashMap<String,Object> data;
+    public String data;
```

Comment:
This seems like a breaking change, we should look into what spec change caused this